backend/mempool-config.sample.json

```diff
@@ -15,9 +15,9 @@
     "INDEXING_BLOCKS_AMOUNT": 11000,
     "PRICE_FEED_UPDATE_INTERVAL": 600,
     "USE_SECOND_NODE_FOR_MINFEE": false,
-    "EXTERNAL_ASSETS": [
-      "https://raw.githubusercontent.com/mempool/mining-pools/master/pools.json"
-    ],
+    "EXTERNAL_ASSETS": [],
```

Comment:
Since this resource is already being fetched & served with https://github.com/mempool/mempool/blob/master/backend/src/tasks/pools-updater.ts, fetching same resource with https://github.com/mempool/mempool/blob/master/backend/src/sync-assets.ts would be useless